src/auth/utils.ts

```diff
@@ -38,6 +38,10 @@ const getAllErrorMessages = (error: ApolloError) => {
           });
         }
       });
+    } else if (networkErrors?.result?.errors.length > 0) {
+      networkErrors.result.errors.forEach(({ message }) => {
+        errorMessages.push(message);
+      });
     } else {
       errorMessages.push(networkErrors.result.errors.message);
     }
```

Comment:
Do I understand it correctly, the `networkErrors.result.errors` is the array with many error objects or the single error object, right?

Reply:
@orzechdev there was some issue with rebasing, fixed now